daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-multi-app.md

```diff
@@ -38,62 +39,37 @@ Similarly, all app IDs must use the same actor state store.
 Finally, the target app ID must have the activity or child workflow defined, otherwise the parent workflow will retry indefinitely.
 {{% /alert %}}
 
-## Multi-application activity examples
-
-The following examples show how to execute activities on different target app IDs.
-
-{{< tabpane text=true >}}
-
-{{% tab "Go" %}}
+{{% alert title="Important Limitations" color="warning" %}}
+- **Only some SDKs support multi-application workflows** - Multi-application workflows are dependent on the specific SDK implementation. Currently only available in the Java (partially) and Go SDKs. Other SDKs (Python, .NET, JavaScript) do not support any multi-application workflows features at this time.
```

Comment:
Can you clarify what partially means here in Java? Is this saying it can only call remote activities and not remote child workflows?

Reply:
Also are there any other examples of samples for Java and Go in the SDK repos that we can point to and put into a next steps section at the end of this article?